libqtile/widget/mpris2widget.py

```diff
@@ -110,9 +110,9 @@ def update(self, interface_name, changed_properties, invalidated_properties):
             if self.stop_pause_text is not None:
                 self.is_playing = False
                 self.displaytext = self.stop_pause_text
-            elif olddisplaytext:
+            elif self.displaytext:
                 self.is_playing = False
-                self.displaytext = 'Paused: {}'.format(olddisplaytext)
+                self.displaytext = 'Paused: {}'.format(self.displaytext)
```

Comment:
Since `self.displaytext` is cleared earlier in the function, this would simply yield `Paused: `.

Reply:
It actually may not if the metadata is changed.

Reply:
Actually, it might work if the metadata is still there.

Reply:
> Actually, it might work if the metadata is still there.

Yeah, and if `self.displaytext` is cleared, this `elif` will not be hit, and it will yield `Paused` in the following `else`.